GetandTake/Services/Concrete/CategoryManager.cs

```diff
@@ -26,10 +27,39 @@ public async Task UpdateAsync(int categoryId, Category category)
             _repository.Update(category);
         }
     }
-    
-    public async Task<List<Category>> GetAllAsync() => 
+
+    public async Task<List<Category>> GetAllAsync() =>
         await _repository.GetItemsAsync();
 
-    public async Task<Category> GetByIdAsync(int categoryId) => 
+    public async Task<Category> GetByIdAsync(int categoryId) =>
         await _repository.GetAsync(category => category.CategoryID == categoryId);
-}
+
+    public async Task UploadImage(IFormFile file, int id)
+    {
+        var category = await GetByIdAsync(id);
+       var category = await GetByIdAsync(id);
+       
+        if (category.ImagePath == null)
+        {
+              throw new Exception(...);
```

Comment:
`...` is not valid, I advised you just a basic idea, please, update exception message with appropriate description (same case for `EditImage`)